dockstore-webservice/src/main/resources/migrations.xml

```diff
@@ -23,9 +23,9 @@
     <include file="migrations.test.confidential1.xml" relativeToChangelogFile="true"/>
     <include file="migrations.test.confidential2.xml" relativeToChangelogFile="true"/>
     <include file="migrations.test.xml" relativeToChangelogFile="true"/>
+    <include file="migrations.test.add_test_tools.xml" relativeToChangelogFile="true"/>
```

Comment:
Just curious, why did you need to move this?

Reply:
See addition to description.